CHANGELOG.md

```diff
@@ -1,3 +1,82 @@
+# 1.8.0-alpha03 (February 2025)
+
+_Changes since 1.8.0-alpha02_
+
+## Highlights
+
+### Multiple Platforms
+
+- [Compose Multiplatform codebase is fully migrated to K2](https://github.com/JetBrains/compose-multiplatform-core/pull/1778). Please note that native and web klibs can be consumed only with Kotlin 2.1.0 or newer. Also, due to underlying changes in the compiler plugin, it's better to recompile libraries against the new version. Please let us know if you find any compatibility issues during this migration
+- [Implement multiplatform `BackHandler` and `PredictiveBackHandler`](https://github.com/JetBrains/compose-multiplatform-core/pull/1771). And use them in material3 widgets and androidx-navigation library
+
+## Breaking Changes
+
+### Multiple Platforms
+
+- [IdlingResource](https://developer.android.com/reference/kotlin/androidx/compose/ui/test/IdlingResource) inteface was moved from commonMain to android and desktop source sets. The related experimental methods of [ComposeUiTest](https://developer.android.com/reference/kotlin/androidx/compose/ui/test/ComposeUiTest) were moved too. They are not available for Web and iOS anymore. Consider using [waitUntil function][(https://developer.android.com/reference/kotlin/androidx/compose/ui/test/ComposeUiTest#waitUntil(kotlin.String,kotlin.Long,kotlin.Function0)) as an alternative](https://github.com/JetBrains/compose-multiplatform-core/pull/1822). Note: it's a breaking change only for Web and iOS, but not for Desktop and Android
+
+### iOS
+
+- [Experimental classes CupertinoScrollDecayAnimationSpec and CupertinoOverscrollEffect are removed from public API](https://github.com/JetBrains/compose-multiplatform-core/pull/1806)
+
+## Features
+
+### Multiple Platforms
+
+- [Adopt a new Clipboard interface with suspend functions, which work correctly on all targets including Web](https://github.com/JetBrains/compose-multiplatform-core/pull/1796). The ClipboardManager was deprecated because it was not possible to correctly implement it for Web
```

Comment:
```suggestion
- [Adopt a new `Clipboard` interface with suspend functions, which work correctly on all targets including Web](https://github.com/JetBrains/compose-multiplatform-core/pull/1796). The `ClipboardManager` was deprecated because it was not possible to correctly implement it for Web
```